parsons/action_network/action_network.py

```diff
@@ -1139,6 +1412,34 @@ def get_person_signature(self, person_id, signature_id):
         """
         return self.api.get_request(f"people/{person_id}/signatures/{signature_id}")
 
+    def create_signature(self, petition_id, data):
+        """
+        `Args:`
+            petition_id:
+               The unique id of the petition
+            data:
+               The data for creating the signature
+        `Returns:`
+            A JSON with the created signature entry
+        """
+        return self.api.post_request(f"petitions/{petition_id}/signatures", data)
+
+    def update_signature(self, petition_id, signature_id, data):
```

Comment:
Why is this one "data" and not "payload"?

Reply:
You're right, didn't see it! I changed it to payload :)